cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -96,6 +99,10 @@ open class Node @Inject constructor(private val project: Project) {
     @get:Input
     var extraConfig: Map<String, Any> = emptyMap()
 
+    @get:Optional
+    @get:Input
+    var dockerConfig: Map<String, Any> = emptyMap()
```

Comment:
`Node` is shared between `Cordform` and `Dockerform`. Does the `dockerConfig` _need_ to be part of `Node`? Could you put it on `Dockerform` and then somehow update the `extraConfig` with the contents of `dockerConfig` at an appropriate time?

Reply:
Moved into the Dockerform to be added at a later time